### ISSUE TITLE
Issue on page /content/recipes/accessibility.html

### ISSUE THREAD

**Issue by Philip-Gribbon** [FAIRplus/the-fair-cookbook#303]
In general, the two current recipes deal with process of Access solely from the perspective of getting the right file transfer software running / making sure firewalls etc are set up. This may have only a limited appeal to audiences

More interesting might be recipes on:
1) How to set up an access procedure for clinical data which meets GDPR and privacy needs. Include exemplar processes + software tools to consider
2) How do I best offer access to verified internal users whilst restricting  access to externals/non-authorised in a resource effective manner using open source s tools
3) How will tools such as face recognition / biometrics etc be use to access personal / secure data in the future. What are the pitfalls in this approach

**Comment by proccaserra** [comment]
@Philip-Gribbon I agree with your assessment about the limited interest. If we have expertise on any of the topics you list in this issue, recipes may be added. Adding squad leads to this issue
